src/Microsoft.ML.Mkl.Components/MklComponentsCatalog.cs

```diff
@@ -46,7 +46,7 @@ public static class MklComponentsCatalog
         }
 
         /// <summary>
-        /// Predict a target using a linear regression model trained with the <see cref="OlsTrainer"/>.
+        ///  Creates a <see cref="OlsTrainer"/>, which predicts a target using a linear regression model with advanced options.
         /// </summary>
```

Comment:
let's keep the pattern consistent:
Create a cref **with advanced options**, short description.